projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -1669,6 +1680,22 @@ protected function filter_non_public_posts( array $related_posts ) {
 		return $filtered;
 	}
 
+	/**
+	 * Generates the author byline for the related post.
+	 *
+	 * @param int $post_id - the post ID.
+	 * @uses get_post_field, get_the_author_meta
+	 * @return string
+	 */
+	protected function generate_related_post_display_author( $post_id ) {
+		$post_author         = get_post_field( 'post_author', $post_id );
+		$author_display_name = get_the_author_meta( 'display_name', $post_author );
+		if ( isset( $author_display_name ) && ! empty( $author_display_name ) ) {
```

Comment:
```suggestion
		if ( ! empty( $author_display_name ) ) {
```
I guess `$author_display_name` is always set by the point 😄